Classes/Controller/SettingsController.php

```diff
@@ -177,4 +181,13 @@ private function validateApiCalls(SettingsDTO $openAi, SettingsDTO $stabilityAi,
         $altTextAi->validateClientApiKey();
         $summAi->validateClientApiKey();
     }
+
+    private function extractNewsContentTypes(): array
+    {
+        $availableNewsContentTypes = [];
+        foreach ($GLOBALS['TCA']['tt_content']['columns']['CType']['config']['items'] as $cType) {
+            $availableNewsContentTypes[] = $cType[1];
+        }
```

Comment:
Currently, dividers (`--div--`) are also taken into account. As this does not make sense at all, they should be skipped like this:

```suggestion
        foreach ($GLOBALS['TCA']['tt_content']['columns']['CType']['config']['items'] as $item) {
        	$cType = $item[1];

            if ($cType !== '--div--') {
	            $availableNewsContentTypes[] = $cType;
	        }
        }
```